src/Section/Card/Card.js

```diff
@@ -0,0 +1,52 @@
+import "./Card.css";
+import { useState } from "react";
+import logo from "../../assets/logo.svg";
+import star from "../../assets/star.svg";
+import kebab from "../../assets/kebab.svg";
+
+const defalutValue = {
```

Comment:
[중요도: 중간] 이런 변하지 않는 상수의 경우는 UPPER_SNAKE_CASE로 해주세요. 이름도 조금 구체적으로 적으면 좋겠군요
```suggestion
const DEFAULT_CARD_VALUE = {
```